certbot/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 * We no longer publish our beta Windows installer as was originally announced
   [here](https://community.letsencrypt.org/t/certbot-discontinuing-windows-beta-support-in-2024/208101).
+* Switch to using docker compose v2, as Azure's images dropped v1.
```

Comment:
we can keep it if ya want, but i'd suggest removing this since this isn't user facing and just affects our internal ci stuff